cherrypy/test/test_params.py

```diff
@@ -0,0 +1,34 @@
+import cherrypy
+from cherrypy.test import helper
+
+
+class ParamsTest(helper.CPWebCase):
+    @staticmethod
+    def setup_server():
+        class Root:
+            @cherrypy.expose
+            @cherrypy.tools.params(exception=ValueError)
+            def resource(self, limit=None, sort=None):
+                return '{0}: {1}'.format(limit, type(limit).__name__)
+            # for testing on Py 2
```

Comment:
It's a `SyntaxError`, so it will have to be an `exec` statement like in `test_objectmapping`.